src/playlist/playlist.cpp

```diff
@@ -98,6 +98,46 @@ const int Playlist::kUndoItemLimit = 500;
 const qint64 Playlist::kMinScrobblePointNsecs = 31ll * kNsecPerSec;
 const qint64 Playlist::kMaxScrobblePointNsecs = 240ll * kNsecPerSec;
 
+// kColumns provides names to enum Columns.
+// When the playlist column is editable those names are used as names
+// for taglib writing to file. Names are caseinsensitive and often
+// 3rdparty/taglib/.../id3v2frame translations is base for naming used.
+// When playlist enum columns is changed the list below has to be adapted
+// accordingly. Lowercase written names miss a taglib equivalent and are
+// used for internal informations.
+// When adding lines to column enumeration: Try to add mappable taglib
+// properties.
+const QStringList Playlist::kColumns = QStringList() << "TITLE"
+                                                     << "ARTIST"
+                                                     << "ALBUM"
+                                                     << "ALBUMARTIST"
+                                                     << "COMPOSER"
+                                                     << "LENGTH"
+                                                     << "TRACKNUMBER"
+                                                     << "DISCNUMBER"
+                                                     << "DATE"
+                                                     << "GENRE"
+                                                     << "BPM"
+                                                     << "bitrate"
+                                                     << "samplerate"
+                                                     << "filename"
+                                                     << "basefilename"
+                                                     << "filesize"
+                                                     << "filetype"
+                                                     << "ctime"
+                                                     << "mtime"
+                                                     << "rating"
+                                                     << "playcount"
+                                                     << "skipcount"
+                                                     << "lastplayed"
+                                                     << "score"
+                                                     << "COMMENT"
+                                                     << "source"
+                                                     << "MOOD"
+                                                     << "performer"
+                                                     << "GROUPING"
+                                                     << "ORIGINALDATE";
```

Comment:
A static GetColumnName method with a case statement is probably less error prone.

Reply:
And if the strings are only for tags, maybe that method should be in TagReaderClient.

Reply:
QStringList Song::kColumns  was taken as example. QStringList might cause more runtime overhead but readability is much better to name enumeration items.